pypeit/scripts/ql.py

```diff
@@ -0,0 +1,200 @@
+"""
+Script for quick-look reductions for Multislit observations.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+import os
+import time
+
+import numpy as np
+
+from astropy.table import Table
+
+from pypeit import utils
+from pypeit.scripts import run_pypeit
+from pypeit import par, msgs
+from pypeit import pypeitsetup
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit.core import setup
+from pypeit.core import quicklook
+from pypeit.scripts import scriptbase
+from pypeit.spectrographs import available_spectrographs
+
+from IPython import embed
+
+class QL(scriptbase.ScriptBase):
+
+    @classmethod
+    def get_parser(cls, width=None):
+        parser = super().get_parser(description='Script to produce quick-look multislit PypeIt reductions', width=width)
+        parser.add_argument('spectrograph', type=str,
+                            help='A valid spectrograph identifier: {0}'.format(
+                                 ', '.join(available_spectrographs)))
+        parser.add_argument('--rawfile_list', type=str, 
+                            help='File providing raw files to reduce including their path(s)')
+        parser.add_argument('--full_rawpath', type=str, 
+                            help='Full path to the raw files. Used with --rawfiles or --extension')
```

Comment:
There is no `--extension` argument as referenced in the help string, but there is `--raw_extension`